src/Broker/ActiveMq/Options.php

```diff
@@ -43,23 +43,27 @@ public function __construct(array $options = [])
         }
     }
 
+    #[\ReturnTypeWillChange]
```

Comment:
add correct return types for all [\ReturnTypeWillChange] because we don't need more php5 support if the composer.json requires minimum php8

Reply:
This would be a BC. 
We should stay compatible with the upstream as good as possible.